all/core/state.py

```diff
@@ -291,7 +304,7 @@ def as_output(self, tensor):
         return tensor.view((*self.shape, *tensor.shape[1:]))
 
     def apply_mask(self, tensor):
-        return tensor * self.mask.unsqueeze(-1)
+        return tensor * self.mask.unsqueeze(-1)  # pylint: disable=no-member
```

Comment:
Don't need the pylint disable because we change the linter

Reply:
Removed